demisto_sdk/commands/common/tests/dependencies_test.py

```diff
@@ -46,6 +46,553 @@ def make_integration_feed(integration: Integration):
         integration.yml.update({'feed': True})
 
 
+# Playbook class helper function
+def get_new_task_number(playbook: Playbook):
+    playbook_tasks = list(playbook.yml.read_dict().get('tasks').keys())
+
+    if playbook_tasks:
+        return max([int(task_num) for task_num in playbook_tasks]) + 1
+    return 0
+
+
+def update_tasks_in_playbook(playbook: Playbook, task_num: int, task: dict):
+    tasks = playbook.yml.read_dict().get('tasks')
+
+    if task_num > 0:
+        try:
+            tasks.get(str(task_num - 1)).get('nexttasks').get('#none#').append(str(task_num))
```

Comment:
why -1

Reply:
This section connects the last task that exists in the playbook to the new task.
For example:
We have 3 tasks already in the playbook, so we want to add a task with id '4', then we want to connect between task '3' and task '4', otherwise, that will fail validation.

Reply:
Adding a comment to that too